app.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const { categoriesHandler } = require('./controllers/games.controller');
+
+const app = express();
+
+app.use(express.json());
+
+app.all('/api/categories', categoriesHandler);
```

Comment:
This should be app.get, this eliminates us having to check for the req.method further on too. Using app.get is already checking that the method is get, same for app.patch, app.delete etc